### DISCUSSION TITLE
is Next.js supported ?

### DISCUSSION THREAD

**Discussion by rishabh1811** [fwouts/previewjs#123] [General]
**No component was found at offset 602** error. I am using a next.js project with tailwind

**Comment by lamhungypl** [comment]
Seem not work (Next 12.0.7, Tailwind 3.0.8)
Minimal demo:
```html
<div className="w-20 h-20 border border-black"></div>
```

**Comment by fwouts** [comment]
Hi there, yes Next.js with Tailwind should be supported, but it sounds like you're running into a bug.

Would either of you be able to provide a sample repo? Thank you :)

Also which version of the extension are you using?

**Comment by mtrogers** [comment]
I'm having trouble with it as well; the props passed from getStaticProps don't seem to be read by preview

**Reply by fwouts** [reply]
Hi @mtrogers, just to clarify, Preview.js only runs your component, it doesn't execute Next-specific callbacks so you need to pass the props you'd like to preview to your component.

Does that make sense?
